### ISSUE TITLE
peer review

### ISSUE THREAD

**Issue by churningbutter** [anGie44/ORIE4741Project-HCMST#2]
strengths:
interesting data set choice
this kind of knowledge and patterns is useful not just for relationships - things like connections and friendships can be thought about the same way
lot of popular companies that use this kind of data
weaknesses:
where is the data coming from?
feel like even though you will find patterns, getting along with a person is more of a personality/whatever(not sure how to phrase this exactly) rather than based on data and numbers
basically every person/relationship will be different even though some things are similar
